tests/App/Resources/config/config.yaml

```diff
@@ -17,8 +17,25 @@ framework:
     csrf_protection:
         enabled: true
     templating:
-        engines: [twig]
+        engines: ['twig']
 
 twig:
     exception_controller: null
     strict_variables: '%kernel.debug%'
+    paths:
+        - '%kernel.project_dir%/Resources/views'
+
+fos_rest:
+    param_fetcher_listener: true
+    body_listener: true
+    format_listener:
+        enabled: true
+        rules:
+            priorities: 'json'
```

Comment:
Looks like this makes every request as `json`, should it be scoped to `^/api`?

Reply:
Thanks for catch this. I have to fix it in whole PRs. Only in this we have functional test.